### ISSUE TITLE
Documentation with release examples, rather than master branch examples

### ISSUE THREAD

**Issue by svincen2** [kubernetes-client/java#643]
Maybe indicate in the master branch documentation that if you're using a release in Maven, the documentation you should be looking at is on a specific tag, not the master branch.  And by documentation, I mean the examples, since that's the only documentation you have for the core logic (other than digging through the source code).  I spent two hours trying to understand why V1Patch doesn't exist, when it's in the patch example. 
Or, have the examples link take you to the latest release examples, rather than examples that will only work if you're *not* using Maven, and rather building the jar from master, or depending on master branch directly.  I image most people are using Maven by now.

**Comment by yue9944882** [comment]
we should probably comment supported version on those example..

```
(5.0.0+) PatchExample: Patch resource objects in various supported patch formats, equal to kubectl patch.
```

**Comment by brendandburns** [comment]
yeah, that's a good idea, probably link to the tag from the Readme table too. 

@svincen2 we'd love a PR to improve the docs, thanks!

**Comment by yue9944882** [comment]
/assign

will send an update soon

**Comment by yue9944882** [comment]
/close

fixed by #648
